tests/Api/Admin/AvatarImageTest.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Api\Admin;
+
+use Sylius\Component\Core\Model\AdminUserInterface;
+use Sylius\Tests\Api\JsonApiTestCase;
+use Sylius\Tests\Api\Utils\AdminUserLoginTrait;
+use Symfony\Component\HttpFoundation\Response;
+
+final class AvatarImageTest extends JsonApiTestCase
+{
+    use AdminUserLoginTrait;
+
+    /** @test */
+    public function it_gets_an_avatar_image(): void
+    {
+        $fixtures = $this->loadFixturesFromFiles(['authentication/api_administrator.yaml', 'avatar_image.yaml']);
+        $header = array_merge($this->logInAdminUser('api@example.com'), self::CONTENT_TYPE_HEADER);
+
+        /** @var AdminUserInterface $adminUser */
+        $adminUser = $fixtures['admin'];
+
+        $this->client->request(
+            method: 'GET',
+            uri: sprintf('/api/v2/admin/administrators/%s/avatar-image', $adminUser->getId()),
+            server: $header,
+        );
+
+        $this->assertResponse(
+            $this->client->getResponse(),
+            'admin/avatar_image/get_an_avatar_image',
```

Comment:
File names can be simplified so that they are not too long, we do so, for example, with the file names of the behat features:
```suggestion
            'admin/avatar_image/get_avatar_image',
```